packages/bezier-react/src/components/AlphaStack/AlphaStack.tsx

```diff
@@ -0,0 +1,74 @@
+/* External dependencies */
+import React, {
+  forwardRef, useMemo,
+} from 'react'
+import type { Ref } from 'react'
+
+/* Internal dependencies */
+import { flex } from '~/src/components/Stack/util'
+import * as Styled from './AlphaStack.styled'
+import type { AlphaStackProps } from './AlphaStack.types'
```

Comment:
```suggestion
import type { AlphaStackProps } from './AlphaStack.types'
import * as Styled from './AlphaStack.styled'
```

린트 규칙이 있진 않으나, 스타일 관련 파일을 맨 아래 작성하는 편입니다([내부 팀 컨벤션 문서](https://www.notion.so/channelio/Import-Modules-30436d844cdb45ada716cb84f80ec28e?pvs=4#dbb43a8b03c946459a1833e9d3b044c3))